_i18n/en/resources/user-guides/verification-allos-advanced.md

```diff
@@ -116,58 +140,73 @@ gpg:          There is no indication that the signature belongs to the owner.
+The steps for both Linux and Mac are the same. From a terminal, get the
+`SHA256` hash of your downloaded Monero binary. As an example this guide
+will use the `Linux, 64bit` GUI binary. Substitute
+`monero-gui-linux-x64-v0.15.0.1.tar.bz2` with the name of the binary that
+you downloaded in [section 4.1](#41-get-monero-binary).
 
-```
-shasum -a 256 monero-linux-x64-v0.15.0.1.tar.bz2
-```
+``` shasum -a 256 monero-linux-x64-v0.15.0.1.tar.bz2 ```
 
-The output will look like this, but will be different for each binary file. Your `SHA256` hash should match the one listed in the `hashes.txt` file for your binary file.
+The output will look like this, but will be different for each binary
+file. Your `SHA256` hash should match the one listed in the `hashes.txt`
+file for your binary file.
 
-```
-8d61f992a7e2dbc3d753470b4928b5bb9134ea14cf6f2973ba11d1600c0ce9ad  monero-linux-x64-v0.15.0.1.tar.bz2
-```
+``` 8d61f992a7e2dbc3d753470b4928b5bb9134ea14cf6f2973ba11d1600c0ce9ad
+monero-linux-x64-v0.15.0.1.tar.bz2 ```
 
-If your hash **DOES** match, then you are finished with the guide! You can extract the files and install.
+If your hash **DOES** match, then you are finished with the guide! You can
+extract the files and install.
 
-If your hash **DOES NOT** match, **DO NOT CONTINUE.** Instead delete the binary you downloaded and go back to [section 4.1](#41-get-monero-binary).
+If your hash **DOES NOT** match, **DO NOT CONTINUE.** Instead delete the
+binary you downloaded and go back to [section 4.1](#41-get-monero-binary).
 
 ### Binary Verification on Windows
 
-From a terminal, get the `SHA256` hash of your downloaded Monero binary. As an example this guide will use the `Windows, 64bit` GUI binary. Substitute `monero-gui-win-x64-v0.15.0.1.zip` with the name of the binary that you downloaded in [section 4.1](#41-get-monero-binary).
+From a terminal, get the `SHA256` hash of your downloaded Monero binary. As
+an example this guide will use the `Windows, 64bit` GUI binary. Substitute
+`monero-gui-win-x64-v0.15.0.1.zip` with the name of the binary that you
+downloaded in [section 4.1](#41-get-monero-binary).
 
-```
-certUtil -hashfile monero-gui-win-x64-v0.15.0.1.zip SHA256
-```
+``` certUtil -hashfile monero-gui-win-x64-v0.15.0.1.zip SHA256 ```
 
-The output will look like this, but will be different for each binary file. Your `SHA256` hash should match the one listed in the `hashes.txt` file for your binary file.
+The output will look like this, but will be different for each binary
+file. Your `SHA256` hash should match the one listed in the `hashes.txt`
+file for your binary file.
 
-```
-SHA256 hash of file monero-gui-win-x64-v0.12.0.0.zip:
-4b 9f 31 68 6e ca ad 97 cd b1 75 e6 57 4b f3 07 f8 d1 c4 10 42 78 25 f4 30 4c 21 da 8a ac 18 64
-CertUtil: -hashfile command completed successfully.
-```
+``` SHA256 hash of file monero-gui-win-x64-v0.12.0.0.zip: 4b 9f 31 68 6e ca
```

Comment:
Will these line breaks show up on the website?

Reply:
It won't. The structure of the guide should remain the same.

Reply:
The netlify preview looks broken due to the line breaks: https://deploy-preview-1639--barolo-time-757cf9.netlify.app/resources/user-guides/verification-allos-advanced.html

Reply:
I see it now. I haven't noticed it using my local markdown previewer. Looking into it. The structure got messed up by the first commit wrapping the .md file.

Reply:
So, for some reason there is no way to avoid wrapping usnig the the `po4a-*` tools. I had to create a configuration file and a workflow for po4a to fix this problem. I have the config file saved locally, i'll tweak it to work for all docs and then i'll PR.

There should be no more problems caused by unexpected wrapping in these files.